src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1416,9 +1430,16 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 	else
 		return NULL;
 
-	if (vector_size == 256 || vector_size == 512)
-		return NULL; 
-		
+	if (vector_size == 256 || vector_size == 512) {
+		if (id == SN_get_IsHardwareAccelerated ) {
```

Comment:
For RyuJIT, we have a fallback that handles any unrecognized `get_IsHardwareAccelerated` and `get_IsSupported` APIs for `System.Numerics`, `System.Runtime.Intrinsics`, and `System.Runtime.Intrinsics.*` to ensure they can be treated as `constant false`. Each namespace is being handled a little bit differently since `get_IsSupported` for some namespaces needs to fallback to user-code instead.

Would it be a good idea to similarly make this general-purpose. Notably `vector_size == 64` should also be `false` on `x86/x64` for example?

Reply:
Reverted this change to rely on fallback for get_IsHardwareAccelerated.